.setup/travis/autograder.sh

```diff
@@ -68,10 +68,7 @@ git clone 'https://github.com/Submitty/AnalysisTools' ${SUBMITTY_INSTALL_DIR}/GI
 
 pushd /usr/local/submitty/GIT_CHECKOUT_AnalysisTools
 
-
-git checkout -b  VERSION_2_1  v0.2.1
-stack --allow-different-user --install-ghc --copy-bins build
-
+./build.sh v0.2.1
```

Comment:
We only need to checkout the repo here as we INSTALL_SUBMITTY.sh does `./build.sh v0.2.1` already.